src/common/components/Search/SearchResultList/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+// Components
+import SearchResultItem from './SearchResultItem';
+
+// Styles
+import * as S from '../styles';
+
+// Types
+import { searchResultType } from '../../../../types';
+
+interface SearchResultListProps {
+  searchResults: [searchResultType];
```

Comment:
if you meant array of SearchResultType, it should be written like this:
`searchResults: SearchResultType[];`